src/Mixins/Expectation.php

```diff
@@ -1128,4 +1129,19 @@ public function toBeUuid(string $message = ''): self
 
         return $this;
     }
+
+    /**
+     * Asserts that the value is an anonymous class.
+     *
+     * @return self<TValue>
+     */
+    public function toBeAnonymous(string $message = ''): self
+    {
+        $this->toBeObject();
```

Comment:
Do we only want to check classes? Functions can be anonymous too, so it might be useful for some people who work with anonymous callables, etc. to have support for `ReflectionFunction::isAnonymous()` too.

Reply:
@owenvoke We already have `toBeCallable` expection to check anonymous function. So I think no need

Reply:
`toBeCallable` isn't necessarily anonymous though. 🤔

Reply:
Ah, probably not then. 👍🏻

Reply:
@owenvoke thanks